hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/viewmodel/BlockType.java

```diff
@@ -16,23 +16,33 @@
 
 package com.hedera.mirror.web3.viewmodel;
 
+import com.hedera.mirror.web3.exception.UnsupportedBlockTypeException;
 import org.apache.commons.lang3.StringUtils;
 
 public record BlockType(String name, long number) {
 
+    private static final String HEX_PREFIX = "0x";
+
     public static final BlockType EARLIEST = new BlockType("earliest", 0L);
     public static final BlockType LATEST = new BlockType("latest", Long.MAX_VALUE);
-    public static final BlockType PENDING = new BlockType("pending", Long.MAX_VALUE);
-    private static final String HEX_PREFIX = "0x";
+    public static final BlockType UNSUPPORTED = new BlockType("", 0L);
```

Comment:
Q: should the number differ from EARLIEST?
This will avoid misconception should someone ever compare the block number or use it for inference.
```suggestion
    public static final BlockType UNSUPPORTED = new BlockType("", -1L);
```
or if you make number an Integer (not making an argument for it)
```suggestion
    public static final BlockType UNSUPPORTED = new BlockType("", null);
```

Reply:
UNSUPPORTED is now removed as it is no longer needed after the changes from https://github.com/hashgraph/hedera-mirror-node/pull/6975#discussion_r1351090690.